src/internal/log/field_test.go

```diff
@@ -54,4 +57,17 @@ func TestProto(t *testing.T) {
 
 	Info(ctx, "time", Proto("time", timestamppb.Now()))
 	Info(ctx, "nil time", Proto("time", (*timestamppb.Timestamp)(nil)))
+
+	v := &versionpb.Version{}
+	dynamic := dynamicpb.NewMessage(v.ProtoReflect().Descriptor())
+	dynamic.Set(v.ProtoReflect().Descriptor().Fields().ByName("major"), protoreflect.ValueOf(uint32(3)))
+	raw, err := protoToJSONMap(dynamic)
+	if err != nil {
+		t.Errorf("protoToJSONMap: %v", err)
+	}
+	if diff := cmp.Diff(map[string]interface{}{"major": float64(3)}, raw); diff != "" {
```

Comment:
nit: `interface{}` -> `any`, since you used it in the other file.

Reply:
Done.